temporal-sdk/src/main/java/io/temporal/internal/sync/SyncWorkflow.java

```diff
@@ -139,6 +144,35 @@ public void handleSignal(String signalName, Optional<Payloads> input, long event
         "signal " + signalName, () -> workflowProc.handleSignal(signalName, input, eventId));
   }
 
+  @Override
+  public void handleUpdate(
+      String updateName, Optional<Payloads> input, long eventId, UpdateProtocolCallback callbacks) {
+    runner.executeInWorkflowThread(
+        "update " + updateName,
+        () -> {
+          // Skip validator on replay
+          if (!callbacks.isReplaying()) {
```

Comment:
Is isReplaying not available anywhere else in this scope? It would be nice not to have that on the interface.

Reply:
it is not